metadata_writer_sqlite_helpers.c

```diff
@@ -78,7 +78,7 @@ uint8_t md_writer_helpers_copy_db(char *prefix, size_t prefix_len,
 
     //Length check has already been made when we received command line argument,
     //so I know there is room
-    snprintf(dst_filename, 128, "%s.%s", prefix, mws->parent->output_format);
+    snprintf(dst_filename, 128, "%s_%d.%s", prefix, mws->node_id, mws->parent->output_format);
```

Comment:
This comment serves as a reminder to ourself. We also need to update the metadata exporter script with the new filename pattern.